src/language-js/printer-estree.js

```diff
@@ -5972,15 +5972,27 @@ function printComment(commentPath, options) {
   }
 }
 
-function isJsDocComment(comment) {
+function isIndentableBlockComment(comment) {
+  // /**/
+  if (!comment.value) {
+    return false;
+  }
+
+  // /** foo */
+  if (comment.value.indexOf("\n") === -1) {
+    return comment.value[0] === "*"
```

Comment:
Couldn’t we just return false here? And if so, remove the above check for the empty comment?